frontends/web/src/routes/buy/info.tsx

```diff
@@ -170,17 +196,14 @@ class BuyInfo extends Component<Props, State> {
                                     label={t('buy.info.selectLabel')}
                                     placeholder={t('buy.info.selectPlaceholder')}
                                     options={[{
-                                            text: t('buy.info.selectPlaceholder'),
+                                            text: t(options.length ? 'buy.info.selectPlaceholder' : 'loading'),
                                             disabled: true,
                                             value: 'choose',
                                         },
-                                        ...accounts.map(({ code, name}) => ({
-                                            text: `${name} (${code})`,
-                                            value: code,
-                                        }))]
+                                        ...options]
                                     }
                                     onChange={e => this.setState({ selected: e.target.value})}
-                                    value={selected}
+                                    value={options.length ? selected : 'choose'}
```

Comment:
if you kept it as is, `value={selected}`, would it still work?
I'm confused betwee this and the `text: t(options.length ? 'buy.info.selectPlaceholder' : 'loading')` above. seems like they're doing similar thing.

Reply:
`text: t(options.length ? 'buy.info.selectPlaceholder' : 'loading')` is the text of the first grayed out option, this works if there is no preselected coin.

If there is a preselected coin, i.e. 'btc', the loading message would never show as the dropdown wants to show the option with the value 'btc'. But all accounts are checked asynchronously for if they can be bought, while checking options is an empty array, and value should point to the first grayed out option with the value 'choose' .... actually if there was a coin or token called 'choose' this wouldn't work .

does that make sense?

Reply:
but while the accounts are being checked, there's no way to select anything regardless of whether a coin is preselected, right.

what happens if the value remains as is, `value={selected}`, and `selected` is preset to say `btc`?

Reply:
then it doesn't show loading, but to be honest it happens so fast, I doubt that anyone would ever see the loading text.